Clappr-tvOS/Classes/Base/MediaOption.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+open class MediaOption: Equatable {
```

Comment:
This file is exactly the same on shared target

Reply:
@CristianDRM thanks, I'm going to remove the duplicity.